www/js/model.js

```diff
@@ -287,6 +341,12 @@ define([
 			getAuthorID: function(/*string*/ id){
 				return id;
 			},
+			getParent: function(/*string*/ id){
+				return this.getNode(id).parentNode;
+			},
+			getDynamic: function(/*string*/ id){
+				return this.getNode(id).dynamic;
+			},
```

Comment:
this was called accumulator in code as per the discussion with Kurt. The functions for this exist by the name of isAccumulator and setAccumulator.